File: addons/hr_holidays/static/tests/mock_server/mock_models/hr_leave_type.js
import { fields, models } from "@web/../tests/web_test_helpers";

export class HrLeaveType extends models.Model {
    _name = "hr.leave.type";

    id = fields.Integer();
    name = fields.Char();

    _records = [
        {
            id: 55,
            name: "Legal Leave",
        },
        {
            id: 65,
            name: "Unpaid Leave",
        },
    ];
}

File: custom_addons/KSW/KSW_attendance_leave/models/hr_leave_type.py
from odoo import fields, models


class HrLeaveType(models.Model):
    _inherit = 'hr.leave.type'

    is_attendance_issue = fields.Boolean(
        string='Depends on Attendance Issue',
        default=False,
        help='Check this box if this leave type requires linking to '
             'specific attendance records (late, early leave, or absence).',
    )

